lib/src/install/baseline.rs

```diff
@@ -381,12 +379,15 @@ pub(crate) fn install_create_rootfs(
 
     // Initialize the /boot filesystem
     let bootdev = if let Some(bootpn) = boot_partno {
-        Some(findpart(bootpn)?)
+        Some(base_partitions.find_partno(bootpn)?)
     } else {
         None
     };
     let boot_uuid = if let Some(bootdev) = bootdev.as_deref() {
```

Comment:
Remove `as_deref()`; switching from `findpart` to `find_partno` above changes `bootdev` from `Option<String>` to `Option<&Partition>` so it's redundant.